### ISSUE TITLE
Add lightning rounds, strategic play combat option

### ISSUE THREAD

**Issue by smartin015** [Fabricate-IO/expedition-app#40]
Round is a fixed amount of time. If you go over the time, you take a bonus damage amount (up to enemy tier?)

When the round is done, generate a random spread of damage numbers that correspond to a dice roll. 
1-5: 0 damage
5-12: 1 damage
13-17: 2 damage 
18-20: 3 damage

**Comment by smartin015** [comment]
See Asana for whom to notify once this is complete.

**Comment by smartin015** [comment]
Similar to another task: "App combat: 3 different difficulties, easy, normal, hard"

**Comment by toddmedema** [comment]
Closing, duplicate of https://github.com/Fabricate-IO/expedition-app/issues/124
